Doc/howto/argparse.rst

```diff
@@ -468,7 +468,7 @@ verbosity argument (check the output of ``python --help``)::
        print(answer)
 
 We have introduced another action, "count",
-to count the number of occurrences of a specific optional arguments:
+to count the number of occurrences of a specific options:
```

Comment:
The old version has this issue, too: shouldn't this be singular "options", or maybe "occurrences of specific options" (omitting "a")?

Reply:
Yes, that makes sense.